index.js

```diff
@@ -1,130 +1,128 @@
-var _ = require('lodash');
+'use strict';
+
 var Q = require('q');
 var npmKeyword = require('npm-keyword');
 var packageJson = require('package-json');
 var ghGot = require('gh-got');
 var EventEmitter = require('events').EventEmitter;
 
-var UPDATE_INTERVAL_IN_SECONDS = 60 * 60;
+var UPDATE_INTERVAL_IN_SECONDS = 60 * 60 + 1;
```

Comment:
Make it a constant and use that. That way you don't need a comment to describe it.